expo/gbe/views/review_act_view.py

```diff
@@ -19,111 +11,71 @@
-            evaluation.bid = act
-            evaluation.save()
-            return HttpResponseRedirect(reverse('act_review_list',
-                                                urlconf='gbe.urls'))
-        else:
-            return render(request, 'gbe/bid_review.tmpl',
-                          {'readonlyform': [actform, performer],
-                           'reviewer': reviewer,
-                           'form': form,
-                           'actionform': actionform,
-                           'actionURL': actionURL,
-                           'conference': conference,
-                           'old_bid': old_bid,
-                           })
-    else:
-        form = BidEvaluationForm(instance=bid_eval)
-        return render(request,
-                      'gbe/bid_review.tmpl',
-                      {'readonlyform': [actform, performer],
-                       'reviewer': reviewer,
-                       'form': form,
-                       'actionform': actionform,
-                       'actionURL': actionURL,
-                       'conference': conference,
-                       'old_bid': old_bid,
-                       })
+    def post(self, request, *args, **kwargs):
+        self.groundwork(request, args, kwargs)
+        self.form = BidEvaluationForm(request.POST, instance=self.bid_eval)
+        return (self.object_not_current_redirect() or
+                self.post_response_for_form(request))
 
+    def create_action_form(self, act):
+        self.actionform = BidStateChangeForm(instance=act)
 
-def _create_action_form(act):
-    actionform = BidStateChangeForm(instance=act)
-    # This requires that the show be scheduled - seems reasonable in
-    # current workflow and lets me order by date.  Also - assumes
-    # that shows are only scheduled once
-    try:
         start = Show.objects.filter(
-            scheduler_events__resources_allocated__resource__actresource___item=act)[0]
-    except:
-        start = ""
-    q = Show.objects.filter(
-        conference=act.conference,
-        scheduler_events__isnull=False).order_by(
-            'scheduler_events__starttime')
-    actionform.fields['show'] = ModelChoiceField(
-        queryset=q,
-        empty_label=None,
-        label='Pick a Show',
-        initial=start)
-    actionURL = reverse('act_changestate',
-                        urlconf='gbe.urls',
-                        args=[act.id])
-    return actionform, actionURL
+            scheduler_events__resources_allocated__resource__actresource___item=act).first()
+        if start is None:
+            start = ""
+        q = Show.objects.filter(
+            conference=act.conference,
+            scheduler_events__isnull=False).order_by(
+                'scheduler_events__starttime')
+        self.actionform.fields['show'] = ModelChoiceField(
+            queryset=q,
+            empty_label=None,
+            label='Pick a Show',
+            initial=start)
```

Comment:
Something went wrong with this refactor.

Integration test case:
- review an Act as an Act Coordinator
- in the Bid coordinator window: 
   - Accept the act
   - Select a show that is not the first show on the list
   - press submit
- you are returned to the review list - it shows "Accepted" and the selected show
- go back to review
   - look at the Bid Coordinator box - the show you selected is NOT what shows up - that is the bug.

I also verified:
- it works on the webfactional test system
- I restarted my server locally to make sure it wasn't just the caching problem.